src/script/util/DebugUtil.ts

```diff
@@ -195,6 +198,26 @@ export class DebugUtil {
     await proteusService['cryptoClient'].debugBreakSession(sessionId);
   }
 
+  async setTeamSupportedProtocols(supportedProtocols: ConversationProtocol[]) {
```

Comment:
Will enable QA to start testing the feature before TM UI for `supportedProtocols` is ready.